cloudformation/template.yml

```diff
@@ -75,12 +71,23 @@ Parameters:
   Project:
     Type: String
     Default: Raster Vision CloudFormation
-    Description: Name of the project associated with the Batch job
+    Description: Name of the project to associate with new resources
 
   Environment:
     Type: String
     Default: Unknown
     Description: Environment of the project being deployed (e.g. Staging, Development, Production)
+    MinLength: 1
+    AllowedPattern: ^[a-zA-Z0-9]*$
```

Comment:
If the environment name is not purely alphanumeric, the resource naming conventions won't work.